controllers/datadogmonitor/controller_test.go

```diff
@@ -939,3 +940,58 @@ func testAuditMonitor() *datadoghqv1alpha1.DatadogMonitor {
 		},
 	}
 }
+
+func Test_checkMonitorDefSync(t *testing.T) {
+	now := time.Unix(1612244495, 0)
+	metaNow := metav1.NewTime(now)
+	fiveMinAgo := now.Add(-time.Minute * 5)
+	metaFiveMinAgo := metav1.NewTime(fiveMinAgo)
+	m := genericMonitor(12345)
```

Comment:
nit - `1612244495` is set in this function too, this is not obvious from the diff.